bigbluebutton-html5/imports/ui/components/presentation/presentation-toast/presentation-uploader-toast/component.jsx

```diff
@@ -292,21 +292,27 @@ function handleDismissToast(toastId) {
     return toast.dismiss(toastId);
 }
 
-const alreadyRenderedPresList = []
+
+
+const alreadyRenderedPresList = [];
 export const PresentationUploaderToast = ({ intl }) => {
 
 	useTracker(() => {
 		
 		const presentationsRenderedFalseAndConversionFalse = Presentations.find({ $or: [{renderedInToast: false}, {"conversion.done": false}] }).fetch();
-		const convertingPresentations = presentationsRenderedFalseAndConversionFalse.filter(p => !p.renderedInToast )
-		let tmpIdconvertingPresentations = presentationsRenderedFalseAndConversionFalse.filter(p => !p.conversion.done)
-			.map(p => { return {temporaryPresentationId: p.temporaryPresentationId, id: p.id}; })
-		UploadingPresentations.find({}).fetch().filter(p => tmpIdconvertingPresentations.findIndex(pres => pres.temporaryPresentationId === p.temporaryPresentationId || pres.id === p.id) !== -1)
-			.map(p => {
-				return UploadingPresentations.remove({$or: [{temporaryPresentationId: p.temporaryPresentationId }, {id: p.id}]})});
+		
+		const convertingPresentations = presentationsRenderedFalseAndConversionFalse.filter(p => !p.renderedInToast );
+		
+		let toRemove = [];
+		UploadingPresentations.find().fetch().map(p => {
```

Comment:
Here, the major problem is that you check for a subscriptionId, but it isn't there, so the `UploadingPresentation` model deletes the data constantly, and therefore it doesn't show anything in the uploading part of the flow (later on, the presentation is added in the UploadingPresentation model, but without a name because it was deleted and then added again, resulting in not displaying the presentation in the toast for this phase). 

Comment: I think that removing the old processing regarding `tmpIdconvertingPresentations`, and the nested `foreaches` would be tottaly fine, but it should be done more carefully, because it was making sure to only remove the UploadingPresentation data when it had been added to the converting phase, and only then.